lib/openstudio-standards/btap/structure.rb

```diff
@@ -411,19 +411,22 @@ def initialize(model = nil, activity = nil, massive = true)
       iwall_m2  = TBD.facets(cspaces, "surface", "wall").map(&:grossArea).sum
 
       # In OpenStudio, partitions are usually limited to interzone walls between
-      # zones, in order to save on simulation times. Partitions typically absent
-      # from a model include walls surrounding lobbies, stairwells, WCs and
-      # technical rooms, as well as separations between similar rooms (e.g.
-      # multiple, side-by-side enclosed offices, a row of hotel rooms).
-      # Comparing BTAP prototype models and samples of building plans for
-      # similar facilities suggest matching modelled partition m2 (or total
-      # floor m2) as a suitable basis to determine the weight of non-modelled
-      # partitions. As this estimate may be more on the high side for many
-      # prototype models, fixed appliances (e.g. fixtures, counters, doors and
-      # windows) are considered included.
-      # partition_m2 = iwall_m2
-      # partition_m2 = floor_m2 if partition_m2 > floor_m2
-      partition_m2 = floor_m2
+      # zones to save on simulation times. Partitions typically absent from a
+      # model include walls surrounding lobbies, stairwells, WCs and technical
+      # rooms, as well as separations between similar rooms (e.g. multiple,
+      # side-by-side enclosed offices, a row of hotel rooms). Comparing BTAP
+      # prototype models and samples of building plans for similar facilities
+      # suggests matching modelled partition m2 with floor m2 as a suitable
+      # basis to determine the mass of non-modelled partitions. As this estimate
+      # may be more on the high side for many prototype models, fixed appliances
+      # (e.g. fixtures, counters, doors and windows) are considered included.
+      # Adjust for larger spaces, based on building category.
+      case @@data[:category]
+      when "commerce"   then partition_m2 = floor_m2 / 2
+      when "industry"   then partition_m2 = floor_m2 / 4
+      when "recreation" then partition_m2 = floor_m2 / 3
```

Comment:
A deeper dive may be required to adjust non-modelled partition m2 / floor area m2. Best guesstimate for now.